test/compress/issue-208.js

```diff
@@ -1,29 +0,0 @@
-do_not_update_lhs: {
```

Comment:
This is the only file with "extensive" testing for `global_defs` before (there is one in `test/compress/reduce_vars.js` as well).

I was hoping `git` will pick this up as a file rename, but this is basically copied verbatim to `test/compress/global_defs.js` with added `evaluate:true`.